test/others/test_schema.py

```diff
@@ -144,6 +201,25 @@ def test_label_to_dict(text_labels):
     assert l_new.answer.offsets_in_document[0].start == 1
 
 
+def test_equal_table_label(table_label_1, table_label_2):
+    assert table_label_1 == table_label_1
+    assert table_label_1 != table_label_2
+
+
+def test_table_label_to_json(table_label_1):
+    j0 = table_label_1.to_json()
+    l_new = Label.from_json(j0)
+    assert l_new == table_label_1
+    assert l_new.answer.offsets_in_document[0].row == 1
+
+
+def test_table_label_to_dict(table_label_1):
+    j0 = table_label_1.to_dict()
+    l_new = Label.from_dict(j0)
```

Comment:
Same thing for this one, I think it would be worth to have an harcoded dict in here.